content/20.treatments.md

```diff
@@ -517,6 +516,43 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
 This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
 This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.
 
+##### Probiotics
+
+Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” [@doi:10.1038/nrgastro.2014.66]. 
+As a result of the supposed modest benefits of probiotics against common viral infections and evidence that they can modulate the immune response [@doi:10.4082/kjfm.2013.34.1.2; @doi:10.2174/1381612824666180116163411], it has been hypothesized that probiotics may have therapeutic benefits against SARS-CoV-2 [@doi:10.3389/fpubh.2020.00186].
```

Comment:
```suggestion
Some studies suggest modest benefits of probiotics against common viral infections and there is evidence to suggest that they can modulate the immune response [@doi:10.4082/kjfm.2013.34.1.2; @doi:10.2174/1381612824666180116163411], as a result it has been hypothesized that probiotics may have therapeutic benefits against SARS-CoV-2 [@doi:10.3389/fpubh.2020.00186].
```

Reply:
@rando better? They are rocky studies, but they are the ones being touted as potential evidence of their effects by pro-probiotic scientists :/ happy to have a closer look. It was a tough one to phrase properly.